third_party/terraform/tests/resource_bigquery_table_test.go.erb

```diff
@@ -306,6 +332,60 @@ EOH
 `, datasetID, cryptoKeyName, tableID)
 }
 
+<% unless version == 'ga' -%>
+func testAccBigQueryTableRangePartitioning(datasetID, tableID string) string {
+	return fmt.Sprintf(`
+	resource "google_bigquery_dataset" "test" {
+		dataset_id = "%s"
+	}
+
+	resource "google_bigquery_table" "test" {
+		provider   = google-beta
+		table_id   = "%s"
+		dataset_id = "${google_bigquery_dataset.test.dataset_id}"
+
+		range_partitioning {
+			field = "id",
+			range {
+				start    = 1
+				end      = 10000
+				interval = 100
+			}
+		}
+
+		schema = <<EOH
+[
+	{
+		"name": "ts",
+		"type": "TIMESTAMP"
+	},
+	{
+		"name": "city",
+		"type": "RECORD",
+		"fields": [
+	{
+		"name": "id",
```

Comment:
because this field is nested in city, we can't use it in range partitioning and the API returns an error - do you mind top-leveling this field? we don't really care about what this schema is, so it could really just be the id field